breadth_first_search/BreadthFirstSearch.java

```diff
@@ -0,0 +1,81 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+
+//Generic type BreadthFirstSearch implementation
+public class BreadthFirstSearch<T> {
+    // HasMap  of lists for Adjacency List Representation
+    public HashMap<T, ArrayList<T>> adj = new HashMap<T, ArrayList<T>> ();
+   
```

Comment:
@jsroyal Sir, It is about removing the unnecessary whitespaces from blank lines 10 and 25.